steam-market-seller.user.js

```diff
@@ -439,7 +474,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
```

Comment:
That part doesn't make any sense.
You are just delaying ALL by 500ms. Not 500ms before/after each push(). They will fire as before: each one after previous without delay. Right just after 500ms from start.

If you really want to implement delay between every push() action, try this:
```
items.forEach(function(item, index) { // The index of the current element being processed in the array
	setTimeout(function(){
		itemQueue.push(item);
	}, 500*(index)); // Multiply desired beetween-actions-delay by the index
});
```
This will not change the actual forEach() behavior, but instead of almost simultaneous push() firing, there will be many simultaneous setTimeout() with different delays which will mimic beetween-actions-delay.